src/sync/app.hpp

```diff
@@ -55,6 +55,7 @@ class App : public std::enable_shared_from_this<App>, public AuthRequestClient,
         util::Optional<std::string> local_app_name;
         util::Optional<std::string> local_app_version;
         util::Optional<uint64_t> default_request_timeout_ms;
+        util::Optional<std::string> device_id;
```

Comment:
This is not the correct place for this– the other device fields (sdk, platform, etc) need to be on here (the config), but device id is actually associated with the user.

Reply:
changed